modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -210,74 +340,107 @@ Properties set: 8
 Labels added: 4
 ----
 
+`LOAD CSV` supports accessing CSV files via _HTTPS_, _HTTP_, and _FTP_.
+`LOAD CSV` will follow _HTTP_ redirects but for security reasons it won't follow redirects which change the protocol, for example, if the redirect is going from _HTTPS_ to _HTTP_.
+
+[NOTE]
+====
+The file location is relative to the import.
+The config setting `server.directories.import` only applies to a local disc but doesn't to remote URLs.
+====
+
+`LOAD CSV` supports resources compressed with _gzip_ and _Deflate_.
+Additionally `LOAD CSV` supports locally stored CSV files compressed with _ZIP_.
+
 
-[[load-csv-importing-large-amounts-of-data]]
-== Importing large amounts of data
+[[load-csv-large-amounts-of-data]]
+=== Large amounts of data
+
+If the CSV file contains a significant number of rows approaching hundreds of thousands or millions, it is recommended that you serialize the data processing and reduce memory overhead by doing so.
+You can achieve this via link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/subqueries/subqueries-in-transactions/[multiple transactions of subqueries].
+The syntax for this is `+CALL { ... } IN TRANSACTIONS+` which instructs Neo4j to commit a transaction after a number of rows.
+The default is 1000 rows.
+To set a different number of rows for a single transaction, append `+OF X ROWS` to `TRANSACTIONS`, where `X` is the desired number of rows.
```

Comment:
i'm fine with removing `OF x ROWS` in general - it's there if readers follow the link.
i included it because the page originally had an extra example for it :)

removing it poses some problems with the current example though (since 869 lines is < 1000, which is the default). that is secondary of course.